data_management_scripts/cleanup_dataset_matrixtables.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3  # noqa: EXE001
+
+import logging
+import subprocess
+import sys
+from datetime import datetime
+
+import click
+from cpg_utils.config import get_config
+from google.cloud import storage
+
+GENOME_PREFIX = 'mt/'
+EXOME_PREFIX = 'exome/mt/'
+
+
+def get_dataset_mt_timestamps(
+    dataset: str,
+    bucket: storage.Bucket,
+    prefix: str,
+):
+    """Finds the .mt directories and their create times in the seqr-main bucket"""
+    mt_timestamps: dict[str, datetime] | None = {}
+    try:
```

Comment:
Instead of a subprocess call and string parsing, this can be done more cleanly (I think) with cloudpathlib

```
MT_SUFFIXES = [GENOME_PREFIX, EXOME_PREFIX]
bucket = get_config()['storage'][dataset]['default']
for suffix in MT_SUFFIXES:
    mt_folder = to_path(join(bucket, suffix))
    if not mt_folder.exists():
        continue  # do something else
    mt_files = mt_folder.glob('*.mt') # a generator
```

Reply:
the timestamps _should_ be recoverable as UNIX timestamps from the GSPath objects - running a little experiment I can see the `st_mtime` (modified time?) but all other attributes are None... hmm

```
>> all_mts = [each_path / '_SUCCESS' for each_path in mt_files]
>> t = all_mts[0]
>> t
'gs://cpg-acute-care-main/exome/mt/3c9acca7967c89b19386d897d14c1add532ebb_1870-acute-care.mt/_SUCCESS'
>> t.stat()
os.stat_result(st_mode=None, st_ino=None, st_dev='gs://', st_nlink=None, st_uid=None, st_gid=None, st_size=0, st_atime=None, st_mtime=1682459118.557, st_ctime=None)
>> t.stat().st_mtime
1682459118.557
```